ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/AndroidApplicationContext.kt

```diff
@@ -0,0 +1,20 @@
+package com.revenuecat.purchases.ui.revenuecatui.helpers
+
+import android.content.Context
+
+/**
+ * Abstraction around [Context]
+ */
+internal interface ApplicationContext {
+    fun getApplicationName(): String
```

Comment:
We will need to add another accesor to access R strings, but I can do that here later :+1: